src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IServiceProviderExtensions.cs

```diff
@@ -15,5 +16,22 @@ internal static class IServiceProviderExtensions
         {
             return (InterfaceType)sp.GetService(typeof(ServiceType));
         }
+
+        public static IProjectService GetProjectService(this IServiceProvider sp)
+        {
+            var componentModel = sp.GetService(typeof(SComponentModel)) as IComponentModel;
+            if (componentModel == null)
+            {
+                return null;
+            }
+
+            var projectServiceAccessor = componentModel.GetService<IProjectServiceAccessor>();
```

Comment:
>IProjectServiceAccessor [](start = 67, length = 23)

Can't we simply [Import] the accessor and call GetProjectService on it instead of [Import]ing the ServiceProvider and calling this method on it?